data_structures/src/chain.rs

```diff
@@ -538,6 +541,30 @@ pub struct BlockMerkleRoots {
     pub tally_hash_merkle_root: Hash,
 }
 
+/// Block merkle tree roots
+#[derive(Debug, Eq, PartialEq, Clone, Serialize, Deserialize, ProtobufConvert, Default)]
+#[protobuf_convert(pb = "witnet::SuperBlock")]
+pub struct SuperBlock {
+    /// Merkle root of the current Active Reputation Set members
```

Comment:
So here "current" means "after consolidating the block with hash `last_block`"?

Reply:
True, it would now be something like the Active Reputation Set that got anchored into the previous superblock